src/test/kotlin/com/vanniktech/rxriddles/Riddle37Test.kt

```diff
@@ -0,0 +1,36 @@
+package com.vanniktech.rxriddles
+
+import com.vanniktech.rxriddles.solutions.Riddle37Solution
+import com.vanniktech.rxriddles.tools.RxRule
+import io.reactivex.subjects.PublishSubject
+import org.junit.Rule
+import org.junit.Test
+import java.io.IOException
+
+/** Solution [Riddle37Solution] */
+class Riddle37Test {
+  @get:Rule val rxRule = RxRule()
+
+  @Test fun map() {
+    val source = PublishSubject.create<Boolean>()
```

Comment:
no need for a subject here we could just use the normal Observable factory methods